damien/__init__.py

```diff
@@ -31,7 +31,7 @@
 from flask_sqlalchemy import SQLAlchemy
 from sqlalchemy.exc import SQLAlchemyError
 
-__version__ = '1.3'
+__version__ = '1.5'
```

Comment:
@lyttam - should this be `1.4`?

Reply:
@johncrossman No, the dev branch should be on 1.5 (missed this step in the last release). #509 will bump the qa branch to 1.4.